### ISSUE TITLE
Popup render issue in Chrome

### ISSUE THREAD

**Issue by alanshaw** [ipfs-shipyard/ipfs-companion#318]
If the popup receives an update than changes it's height whilst it is animating in then it goes bad:

<img width="360" alt="screen shot 2017-11-21 at 20 07 39" src="https://user-images.githubusercontent.com/152863/33094335-b7c5762a-cef7-11e7-9638-3913bb39e1dc.png">

**Comment by lidel** [comment]
Hm was not able to replicate so far, seems to be specific to Chrome.

Is this the same issue as https://bugs.chromium.org/p/chromium/issues/detail?id=457887 ? 
Its over 1 year old, and they prioritize things oddly (eg. [it took 4 years to fix `background-size`](https://bugs.chromium.org/p/chromium/issues/detail?id=110195)), so we may want to address this on our side.

Perhaps we could detect running in Chrome and producing some kind of  additional, delayed update after initial load?

**Comment by lidel** [comment]
@alanshaw mind checking if changes merged with #336 helped by any chance?

**Comment by jnthnvctr** [comment]
Hi @lidel, unfortunately I believe this is still persisting (I'm also on macOS accessing using Chrome).
Chrome version I'm on is: 63.0.3239.84

This thread had a few work arounds people had found for the bug:  https://bugs.chromium.org/p/chromium/issues/detail?id=428044

**Comment by lidel** [comment]
Thanks @jnthnvctr! @olizilla was able to replicate it in https://github.com/ipfs-shipyard/ipfs-companion/pull/354#issuecomment-356303354.

**tl;dr** it is indeed specific to Chrome on macOS ([potential explanation](https://bugs.chromium.org/p/chromium/issues/detail?id=457887#c4))

I am looking for a volunteer with a Mac to go over potential fixes in #355 :)

**Comment by lidel** [comment]
Seems to be fixed in gurrent `master` – https://github.com/ipfs-shipyard/ipfs-companion/pull/355#issuecomment-368478592
Feel free to reopen if happens again.

**Comment by olizilla** [comment]
@lidel it's happening again on the beta-channel chrome release

<img width="425" alt="screenshot 2018-04-04 13 12 02" src="https://user-images.githubusercontent.com/58871/38306971-249dcd58-380a-11e8-9e5f-a333799437fd.png">

also, i don't have permissions to re-open bugs on this repo, plz can haz?

**Comment by lidel** [comment]
@olizilla try to reopen now :)
